doc/source/users_guide/running-single-points/supported-tower-sites.rst

```diff
@@ -0,0 +1,129 @@
+                        Don't check for input data. Implies --setup-only.
+                        [default: False]
+  --rerun               If the case exists but does not appear to be complete,
+                        restart it. [default: False]
+  --no-batch            Run locally, do not use batch queueing system (if
+                        defined for Machine) [default: False]
+  --run-type            ad, postad, transient
+                        Type of run to do [default: None]
+  --prism               Uses the PRISM reanaylsis precipitation data for the
+                        site instead of the NEON data (only available over
+                        Continental US)
+  --experiment EXPERIMENT
+                        Appends the case name with string for model experiment
+  --run-from-postad     For transient runs only. By default start from
+                        finidat, if this flag is used the postad run must be
+                        available.
+  --neon-version        v1,v2,v3
+                        Neon data version to use for this simulation.
+                        [default: use the latest data available]
+  --xmlchange XMLCHANGE
+                        Any xmlchanges (e.g.,
+                        CLM_CO2_TYPE=constant,CCSM_CO2_PPMV=500) [default:
+                        None]
+
+Logging options:
+
+  -d, --debug           Print debug information (very verbose) to file /glade/
+                        work/tking/ctsm_project/sam_ctsm/CTSM/tools/site_and_r
+                        egional/run_tower.log
+  -v, --verbose         Add additional context (time and file) to log messages
+  -s, --silent          Print only warnings and error messages
+
+
+A `tutorial <https://ncar.github.io/ncar-neon-books/notebooks/NEON_Simulation_Tutorial.html>`_ on running and evaluating data from ``run_tower`` is also available.
+
+.. warning:: Note that the run_tower base case must be of same run type as a requested clone, as described by this `issue ticket <https://github.com/ESCOMP/CTSM/issues/1926>`_.
+
+=========================================
+NEON Tower Single Point Simulations
+=========================================
+
+With this tool, CLM uses gap-filled meteorology from NEON tower sites, the dominant plant species is mapped to the appropriate model plant functional type (PFT), and soil characteristics used in the simulations are updated to match observations from NEON's soil megapits. Gap-filled NEON tower flux data are also available for model evaluation. Additionally, all the commands to run the model are combined into a script that you can easily call from a single line of code.
+
+Currently supported NEON sites include the following: ABBY, BARR, BART, BLAN, BONA, CLBJ, CPER, DCFS, DEJU, DELA, DSNY, GRSM, GUAN, HARV, HEAL, JERC, JORN, KONA, KONZ, LAJA, LENO, MLBS, MOAB, NIWO, NOGP, OAES, ONAQ, ORNL, OSBS, PUUM, RMNP, SCBI, SERC, SJER, SOAP, SRER, STEI, STER, TALL, TEAK, TOOL, TREE, UKFS, UNDE, WOOD, WREF, YELL, all
```

Comment:
I would suggest deleting this, because it'll be annoying to have to keep updated. Instead, provide instructions for the user to find the list of supported sites.